### PR TITLE
add multimodal section to network page

### PR DESCRIPTION
**Pull request by gregbunce**
and add bike blog post link and lrs link to may road update post.

### CODE REVIEW

File: data/transportation/street-network-analysis/index.html [hunk 1]
```diff
@@ -45,3 +51,21 @@ <h3 id="StreetNetwork">Street Network</h3>
     info=page.StreetNetwork
   %}
 </div>
+<div class="grid package">
+  <div class="grid__col grid__col--12-of-12">
+    <h3 id="StreetNetwork">Multimodal Network</h3>
```
Comment (steveoh) [inline data/transportation/street-network-analysis/index.html:56]:
```suggestion
    <h3 id="MultiModalNetwork">Multimodal Network</h3>
```

Comment (gregbunce) [reply data/transportation/street-network-analysis/index.html]: nice!  thx, yup, i just found that, too.  i appreciate it!

**Comment by gregbunce** [comment]
it's still being cranky.  when i click check the netlify logs, "please check the logs" i get this message: "page not found. you might not have permission to see this page."

File: data/transportation/street-network-analysis/index.html [hunk 2]
```diff
@@ -16,6 +16,12 @@
       label: Street Network
   updates:
     - May 20, 2020
+MultimodalNetwork:
+  downloads:
+    - url: https://drive.google.com/uc?id=1DK1eYvl7NAHeL0zdFe8MgwSP2NA9XQcN&export=download
+    label: Multimodal Network
```
Comment (steveoh) [inline data/transportation/street-network-analysis/index.html:22]: issue: your spacing is off here.
```suggestion
      label: Multimodal Network
```

**Comment by gregbunce** [comment]
it looks good to me.  let me know if i missed anything or need to change anything.

File: _posts/2020-05-20-utah-sgid-statewide-roads-data-layer-updates-05202020.md [hunk 3]
```diff
@@ -33,6 +33,10 @@ Data was requested but not received for the following counties:
 
 - **Grand County**
 
+#### UDOT Route System
+
+- Read about the [changes coming to the LRS]("https://drive.google.com/file/d/1t03AWYHeqTjssTSKnzn60ygdIB-nqEAF/view").
```
Comment (steveoh) [inline _posts/2020-05-20-utah-sgid-statewide-roads-data-layer-updates-05202020.md:38]: this link does not work. remove the quotes
```suggestion
- Read about the [changes coming to the LRS](https://drive.google.com/file/d/1t03AWYHeqTjssTSKnzn60ygdIB-nqEAF/view).
```

**Comment by steveoh** [comment]
You're missing an entry on https://deploy-preview-1408--gis-utah-gov.netlify.app/data/transportation. Did you add this to the stewardship tab so it ends up in the sgid index page?

**Comment by gregbunce** [comment]
interesting point.  i can certainly add it.  and in that case, i should add the original network dataset and also the ng911 dataset to the index (stewardship tab).  these would be datasets and not layers (as column C states).  but, you bring up a good point.  it does seem like they should be in the index.  what do you think?  we would have to think about columns B and C, and how to fit these datasets into them.  i like what you're saying.

**Comment by steveoh** [comment]
That would be great. Everything available on our data pages should be in the stewardship sheet. I believe I parse the C column by category and then name so fit it to an existing category in even though it is not in the SGID database. e.g. `TRANPORTATION.NetworkDataset` or something. That will put it in the right group in the sgid-index...

image

**Comment by gregbunce** [comment]
see what you think (no rush).  i added rows 660 and 661.  if that works, I'll also add a record for the ng911 dataset - using that format. https://docs.google.com/spreadsheets/d/11ASS7LnxgpnD0jN4utzklREgMf1pcvYjcXcIcESHweQ/edit#gid=1

**Comment by steveoh** [comment]
Looks good to me. Are you planning to add those items to the table in the transportation index.html page as well?

**Comment by gregbunce** [comment]
doh.  thx!

**Pull request merged**